src/payment.coffee

```diff
@@ -93,6 +85,10 @@ cards = [
   }
 ]
 
+# Store HTML elements (passed onto formatters)
+# for re-use by other methods
+elements = []
```

Comment:
Yeah, its a bit hack-ish.

This change was needed in order to implement ```CardType``` based ```CVC``` length.

Previously, once you bind the inputs, you have no way of getting the ```CardNumber``` input and thus you have to traverse the DOM and guess what the card type might be. The safest approach was to always guess the ```CardType``` from the ```CardNumber``` itself.

By keeping the elements passed during initialization, you can have interaction between the various formatters without resorting to traversing or global variables.